app/services/document_capture_session_result.rb

```diff
@@ -10,11 +10,11 @@
   :failed_back_image_fingerprints,
   :captured_at,
   :selfie_check_performed,
-  :doc_auth_success, :selfie_success,
+  :doc_auth_success, :selfie_status,
   keyword_init: true,
   allowed_members: [:id, :success, :attention_with_barcode, :failed_front_image_fingerprints,
                     :failed_back_image_fingerprints, :captured_at, :selfie_check_performed,
-                    :doc_auth_success, :selfie_success]
+                    :doc_auth_success, :selfie_status]
```

Comment:
this struct is serialized in redis, which means it will get persisted across hosts in a deploy. We need to add some 50-50 logic if we're renaming this, such as allowing both properties to be read, etc etc, then eventually dropping the old name

Reply:
Thanks, I'll make a follow-up ticket to remove the old field and read from both fields in this PR. 👍🏻

Reply:
Follow-up ticket: [LG-12200](https://cm-jira.usa.gov/browse/LG-12200)